src/tests/orders.spec.js

```diff
@@ -0,0 +1,124 @@
+import { expect, test } from "@playwright/test";
+import { ApplicationPages } from "./pages/application.pages.js";
+import { OrdersPage } from "./pages/orders.page.js";
+import { icoNumber, clientName, clientAddress,
+    substitute, contactName, contactPhone, contactEmail,
+    startDate1, endDate1, startDate2, endDate2, startDate3, endDate3,
+    campStudentsNumber, campStudentsAge, campAdultsNumber,
+    natureStudentsNumber, natureStudentsAge, natureAdultsNumber, natureStartTime, natureEndTime,
+    orderConfirmationTitle
+ } from "../fixtures/fixtures.js";
+
+test.describe("Orders", async () => {
+    let application;
+    let order;
+
+    test.describe("Navigation to Orders Page", async () => {
+
+        test.beforeEach(async ({ page }) => {
+            application = new ApplicationPages(page);
+            await application.open();
+        });
+
+        test("User can navigate to Orders for schools from 'For Teachers' menu item", async ({ page }) => {
+            await expect(application.forTeachersMenuItemLocator).toBeVisible();
+            await application.forTeachersMenuItemLocator.click();
+            await expect(application.newOrderLocator).toBeVisible();
+        });
+
+        test("Order form should open upon clicking on 'For Teachers' menu item and user can fill in order details", async ({ page }) => {
+            await application.gotoNewOrder();
+
+            order = new OrdersPage(page);
+            const isOrderFormEnabled = await order.isOrderFormVisibleAndEnabled();
+            await expect(isOrderFormEnabled).toBe(true);
```

Comment:
Await netřeba, dá se použít _toBeTruthy_.
```suggestion
            expect(isOrderFormEnabled).toBeTruthy();
```